src/EnergyPlus/DXCoils.cc

```diff
@@ -511,15 +511,15 @@ void SimDXCoilMultiMode(EnergyPlusData &state,
         // run stage 1 at zero part load to set leaving conditions
         if ((thisDXCoil.NumCapacityStages == 1) || (PartLoadRatio <= 0.0)) {
             // Run stage 1 at its part load
-            PerfMode = DehumidMode * 2 + 1;
-            CalcDoe2DXCoil(state, DXCoilNum, HVAC::CompressorOperation::On, FirstHVACIteration, PartLoadRatio, FanOpMode, PerfMode);
+            PerfMode = (int)DehumidMode * 2 + 1;  // This.  This is not good.  Don't do math on enums.
```

Comment:
Yikes. This is a big no-no. You should not do math on an enumeration. It's a source of future bugs. Someone will have to address this at some point. Probably me, but not in this PR.